docs/user/Curator/index.md

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
Nit: can you change folder name from `Curator` to `curator`? Since this defines the URL